src/requests/addGroup.spec.js

```diff
@@ -32,3 +33,11 @@ test(`create a new tracked link`, async (t) => {
 
   t.is(newGroupId, NEW_GROUP_ID)
 })
+
+test(`GroupNameCharLimitExceeded is thrown when the group name is too long`, async (t) => {
+  const GROUP_NAME = 'hereisareallyreallylonggroupname'
+
+  const error = await t.throwsAsync(addGroup(GROUP_NAME))
```

Comment:
```suggestion
  const error = await t.throwsAsync(
    addGroup(GROUP_NAME),
    GroupNameCharLimitExceeded
  )
```

Reply:
You can pass an error class as the second argument to `throwsAsync`

Reply:
Sure sure